### DISCUSSION TITLE
Gender Identification Following Export of Population Simulation Data

### DISCUSSION THREAD

**Discussion by SayeedMo** [Open-Systems-Pharmacology/Forum#1428] [Questions & Problems]
Hello 

I did a population simulation using PK-SIM based on a population comprised of specific number of males and females. Following export of the simulated results, I was not able to differentiate which of the simulated organisms were male and female. 
The resulting exported csv/excel file identifies each of the virtual human as organism 0,1,2,3...and so and displays the corresponding plasma levels of simulated compound - but there is no gender information in the exported csv file mentioned and thus it is not clear which of the simulated individuals are males or females.

Kindly let me know what is the procedure to include the gender information also in the exported csv file.

Thanks

**Comment by Yuri05** [comment]
You can additionally export the used population to CSV, then you will see all anatomical and physiological properties which correspond to the individual ids.

<img width="500" src="https://user-images.githubusercontent.com/25061876/210855740-a59105b2-89a9-48e8-9cae-872ee30ca4fb.png">

🔽 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 🔽 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 🔽 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 🔽       
 
![grafik](https://user-images.githubusercontent.com/25061876/210855943-c92e08e1-d375-40ad-90f3-9206bf79c8a9.png)
